1-select-tasks/21.txt

```diff
@@ -1,3 +1,11 @@
 Write a SQL statement to get the maximum purchase amount of all the orders.
 
-| Order Id | Maximum Purchase Amount, $ |
\ No newline at end of file
+| Order Id | Maximum Purchase Amount, $ |
+_______________
+
+SELECT TOP(1) [ods].[OrderID], SUM([odsDet].[UnitPrice] * [odsDet].[Quantity] - ([odsDet].[UnitPrice] * [odsDet].[Quantity]) * [odsDet].[Discount]) AS [Maximum Purchase Amount, $]
+FROM [Northwind].[dbo].[Orders] [ods]
+JOIN [Northwind].[dbo].[Order Details] [odsDet]
```

Comment:
usually you don't need to specify db name. Or if you need to specify db name for the whole query do it with "use" statement https://docs.microsoft.com/en-us/sql/t-sql/language-elements/use-transact-sql

Reply:
get it